lib/rucio/tests/temp_factories.py

```diff
@@ -318,8 +322,9 @@ def upload_test_dataset(self, rse_name, scope=None, size=2, nb_files=2):
         self.created_dids.append(did)
         items = list()
         for _ in range(0, nb_files):
+            # TODO : Call did_name_generator
             path = file_generator(size=size)
-            name = os.path.basename(path)
```

Comment:
please see the comment above regarding adding the os.path.basename(path) inside the did_name_generator method :)